bool arm_base_up
bool arm_base_down

bool arm_rot_left
bool arm_rot_right

bool arm_grip_open
bool arm_grip_close
